tests/yaml_configs/abi_test_low_memory.yaml

```diff
@@ -1,12 +1,12 @@
 output_config_type: single_source
 reader_name: abi_netcdf
-filenames: !ENV ${GEOIPS_TESTDATA_DIR}/test_data_noaa_aws/data/goes16/20200918/1950/*
+filenames: !ENV ${GEOIPS_TESTDATA_DIR}/test_data_abi/data/goes16_20200918_1950/*
```

Comment:
We removed all instances of test_data_noaa_aws - those datasets should all exist in test_data_abi, test_data_ahi, etc.